src/teb_local_planner_ros.cpp

```diff
@@ -358,11 +358,18 @@ uint32_t TebLocalPlannerROS::computeVelocityCommands(const geometry_msgs::PoseSt
   // Now perform the actual planning
 //   bool success = planner_->plan(robot_pose_, robot_goal_, robot_vel_, cfg_.goal_tolerance.free_goal_vel); // straight line init
   bool success = planner_->plan(transformed_plan, &robot_vel_, cfg_.goal_tolerance.free_goal_vel);
+
+  // Visualize plan, obstacles and via points
+  planner_->visualize();
```

Comment:
Q: is it possible to add the failed trajectories to other namespace? (e.g `failed_plan`)

Reply:
must be other topic;  trajectories are shown as a pose array, no visual markers

Reply:
would be nice to show them with a different color... :thinking:

Reply:
kinda,,, but it's gonna be rather ugly, so not worth it